Docs (#59)
monarch-initiative/pheval: docs/contributing.md

# Contributing to PhEval

:+1: First of all: Thank you for taking the time to contribute!

> The following is a set of guidelines for contributing to monarch-technical-documentation. 

Author: ```suggestion
The following is a set of guidelines for contributing to the PhEval framework. 
```

These guidelines are not strict rules. Use your best judgment, and feel free to propose 
changes to this document in a pull request.

## Table Of Contents

> - [Contributing to PhEval](#contributing-to-pheval)

Author: ```suggestion
- [Contributing to PhEval](#contributions)
```

  - [Table Of Contents](#table-of-contents)
  - [Code of Conduct](#code-of-conduct)
  - [Guidelines for Contributions and Requests](#guidelines-for-contributions-and-requests)
    - [Reporting problems with the data model](#reporting-problems-with-the-data-model)

<a id="code-of-conduct"></a>

## Code of Conduct

> The monarch-technical-documentation team strives to create a

Author: ```suggestion
The PhEval team strives to create a
```
